manager/forms.py

```diff
@@ -28,6 +29,15 @@ def __init__(self, *args, **kwargs):
         self.fields['recipient_groups'].queryset = RecipientGroup.objects.filter(
             archived=False, preview=False)
 
+    def clean(self):
+        cleaned_data = super(EmailCreateUpdateForm, self).clean()
+        if not cleaned_data.get('recipient_groups') and not cleaned_data.get('segments'):
```

Comment:
Should this be an `or` statement instead of `and` here?

Reply:
We only want to throw the ValidationError if both `recipient_groups` _and_ `segments` are `None`. If one of them has value, then it validates.